lib/casserver/views/login.erb

```diff
@@ -8,6 +8,17 @@
         <% if @message %>
            <p><%= escape_html @message[:message] %></p>
         <% end %>
+
+        <% if request.cookies["login_flash"] %>
+          <p><%= escape_html request.cookies["login_flash"] %></p>
+          <%
+            response.set_cookie("login_flash", {
+              :value => '',
+              :expires => Time.now,
+              :domain => '.beyondz.org'
```

Comment:
On Tue, Apr 21, 2015 at 07:55:58AM -0700, Brian Sadler wrote:
> Is there an existing config value that we can read the domain from?

Nope, since the cookie domain needs to be top level the existing
@domain in there won't work - it will be wrong ons staging.

so it'll have to be a new one